tdrs-frontend/src/components/Reports/FRAReports.jsx

```diff
@@ -0,0 +1,797 @@
+    }
+
+    setSelectedFile(encodedFile)
+  }
+
+  const onSubmit = (e) => {
+    e.preventDefault()
+
+    if (!!error) {
+      return
+    }
+
+    if (!file || (file && file.id)) {
+      setLocalAlertState({
+        active: true,
+        type: 'error',
+        message: 'No changes have been made to data files',
+      })
+      return
+    }
+
+    handleUpload({ file })
+  }
+
+  const formattedSectionName = section.toLowerCase().replace(' ', '-')
+
+  const ariaDescription = file
+    ? `Selected File ${file?.name}. To change the selected file, click this button.`
+    : `Drag file here or choose from folder.`
+
+  return (
+    <>
+      <form onSubmit={onSubmit}>
+        <div
+          className={`usa-form-group ${error ? 'usa-form-group--error' : ''}`}
+        >
+          <label className="usa-label text-bold" htmlFor="uploadReport">
+            {section}
+          </label>
+          <div>
+            {error && (
+              <div
+                className="usa-error-message"
+                id={`${formattedSectionName}-error-alert`}
+                role="alert"
+              >
+                {error}
+              </div>
+            )}
+          </div>
+          <div
+            id={`${formattedSectionName}-file`}
+            aria-hidden
+            className="display-none"
+          >
+            {ariaDescription}
+          </div>
+          <input
+            ref={inputRef}
+            onChange={onFileChanged}
+            id="fra-file-upload"
+            className="usa-file-input"
+            type="file"
+            name={section}
+            aria-describedby={`${formattedSectionName}-file`}
+            aria-hidden="false"
+            data-errormessage={INVALID_FILE_ERROR}
+          />
+          <div style={{ marginTop: '25px' }}>
+            {file?.id ? (
+              <Button
+                className="tanf-file-download-btn"
+                type="button"
+                onClick={handleDownload}
+              >
+                Download {section}
+              </Button>
+            ) : null}
+          </div>
+        </div>
+
+        <div className="buttonContainer margin-y-4">
+          <Button className="card:margin-y-1" type="submit">
+            Submit Report
+          </Button>
+
+          <Button className="cancel" type="button" onClick={handleCancel}>
+            Cancel
+          </Button>
+        </div>
+      </form>
+    </>
+  )
+}
+
+const SubmissionHistory = () => <></>
+
+const ReportTypeSubtext = ({ reportType, reportTypeLabel }) => {
+  let description = ''
+  let aboutLink = ''
+  let templateLink = ''
+
+  switch (reportType) {
+    case 'workOutcomesOfTanfExiters':
+      description =
+        'The Work Outcomes of TANF Exiters report contains the Social Security Numbers (SSNs) of all work-eligible individuals who exit TANF in a given quarter and the dates in YYYYMM format that each individual exited TANF.'
```

Comment:
@reitermb @victoriaatraft has this text been workshopped with end users? just curious what informed this language? im also wondering if we need to specify the format for the date, especially since we dont also include the ssn format.

Reply:
This did appear in the prototypes we used but we weren't specifically focused on content testing of it. I think I might lean slightly toward adding a note to the SSN side mentioning that they must appear without dashes or other punctuation just to act as a mini instruction set for the report. But given that there are instructions elsewhere and the template below that copy I don't think it's crucial!